src/Swift.Runtime/src/Metadata/TypeMetadata.cs

```diff
@@ -124,6 +124,12 @@ public enum TypeMetadataKind {
 public readonly struct TypeMetadata : IEquatable<TypeMetadata> {
     readonly IntPtr handle;
 
+    static TypeMetadata ()
```

Comment:
This is private right? What is the intended usage?

Reply:
static constructor for initializing the cache.